AdventureInnovation/AdventureInnovation/database/seeds/VideosTableSeeder.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class VideosTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('videos')->insert([
```

Comment:
do this

   public function run()
    {
        $user = User::where('email', '=', 'brennan@brennan.com')->firstOrFail();

        $video = new Video();
        $video->Youtube_URL = 'https://www.youtube.com/embed/tgbNymZ7vqY';
        $video->Facebook_URL = 'https://www.facebook.com/canada150th/';
        $video->Twitter_URL = 'https://twitter.com/canada150th?lang=en';
        $video->Instagram_URL = 'https://www.instagram.com/explore/tags/canada150/';
        $video->guide_id = $user->guide->id;

        $video->save();

    }

Reply:
Sorry, wasn't clear.  This just ensures that  we're seeding a specific usesr with those videos

Reply:
Also, the supplied link doesn't seem to embed properly ( ie, just shows up as text when i clear everything out and repopulate the DB).  If i change it to https://youtu.be/tgbNymZ7vqY, it seems to work